packages/hds/src/components/Gauge/Gauge.css.ts

```diff
@@ -0,0 +1,11 @@
+import { style } from '@vanilla-extract/css';
+
+export const guageCss = style({
+  width: '350px',
+  height: '8px',
+  borderRadius: '20px',
+  position: 'relative',
+   transition: 'all 0.25s ease-in-out',
```

Comment:
width 에만 transiton 이 걸리면 되는걸로 보이는데 all 로 할 필요가 없어보여요.
```suggestion
   transition: 'width 0.25s ease-in-out',
```